app/models/operations/load_benefit_market_catalog.rb

```diff
@@ -53,7 +53,7 @@ def load_rating_factors(input)
 
     def load_service_areas(input)
       puts ':: Loading Service Areas ::'
-      files = Dir.glob(File.join(Rails.root, "db/seedfiles/plan_xmls/#{input[:state]}/xls_templates/service_areas", '**', '*.xlsx'))
+      files = Dir.glob(File.join(Rails.root, "db/seedfiles/plan_xmls/#{input[:state]}/xls_templates/service_areas", '**', '*.xls*'))
```

Comment:
you missed this, need to support both file types.

Reply:
Yes it supports both xls and xlsx @nisanthyaganti9 , I have added "xls*"